src/components/Dashboard/TodoList.vue

```diff
@@ -202,6 +218,19 @@
         </v-expansion-panel-header>
 
         <v-expansion-panel-content>
+
+          <div v-if="isCorrection(item)" data-test-class="correction-comment" class="todo-list-detail">
```

Comment:
As in FilingHistoryList, I'm in favour of renaming "item" to "task" here. Up to you.

Reply:
That makes sense to me, no problem!

Reply:
Done and done 👍